hermes/src/state/benchmarks.rs

```diff
@@ -56,22 +55,50 @@ impl TryFrom<BinaryBlob> for Vec<Vec<u8>> {
     }
 }
 
-impl TryFrom<BenchmarkUpdates> for PriceFeedsWithUpdateData {
+
+impl TryFrom<PriceUpdate> for PriceFeedsWithUpdateData {
```

Comment:
I think this definition should be close to PriceUpdate.

nit: I do have a mixed feeling of having them all in the api module. we could entirely remove PriceFeedsWithUpdateData it it was not necessary for v1 api. Maybe let's move it after we deprecate v1 api.

Reply:
hmm should we move it to `types.rs` instead then?